src/models/lineChart.js

```diff
@@ -251,7 +254,7 @@ nv.models.lineChart = function() {
                     .position({left: e.mouseX + margin.left, top: e.mouseY + margin.top})
                     .chartContainer(that.parentNode)
                     .valueFormatter(function(d,i) {
-                        return yAxis.tickFormat()(d);
+                        return d == null ? "N/A" : yAxis.tickFormat()(d);
```

Comment:
I agree the tooltip should handle nulls by default, looking back at this it's fine I think, they can override the valueFormatter option on the guideline tooltip if they want to change it, I was thinking the overridden function would get passed the "N/A" but that's not the case so I think it's fine.